staging/src/k8s.io/apiserver/pkg/server/routes/openapi.go

```diff
@@ -38,9 +38,17 @@ func (oa OpenAPI) Install(c *restful.Container, mux *mux.PathRecorderMux) (*hand
 	if err != nil {
 		klog.Fatalf("Failed to build open api spec for root: %v", err)
 	}
-	openAPIVersionedService, err := handler.RegisterOpenAPIVersionedService(spec, "/openapi/v2", mux)
+
+	openAPIVersionedService, err := handler.NewOpenAPIService(spec)
+	if err != nil {
+		klog.Fatalf("Failed to create OpenAPIService: %v", err)
+		return nil, spec
```

Comment:
a return statement after klog.Fatal is misleading, drop this

Reply:
Done. Thanks.